core/common/src/LocalDate.kt

```diff
@@ -471,7 +473,7 @@ public expect fun LocalDate.plus(value: Int, unit: DateTimeUnit.DateBased): Loca
  * @throws DateTimeArithmeticException if the result exceeds the boundaries of [LocalDate].
  * @sample kotlinx.datetime.test.samples.LocalDateSamples.minus
  */
-public expect fun LocalDate.minus(value: Int, unit: DateTimeUnit.DateBased): LocalDate
+public fun LocalDate.minus(value: Int, unit: DateTimeUnit.DateBased): LocalDate = plus(-value.toLong(), unit)
```

Comment:
it might be a good idea to use parenthesis here, because of this:
```kt
val value = Int.MIN_VALUE
println(  -value.toLong()    ) // 2147483648
println(  -(value.toLong())  ) // 2147483648
println(  (-value).toLong()  ) // -2147483648
```